src/config.py

```diff
@@ -8,6 +8,10 @@
 EMBED_FIELD_VALUE_LIMIT = 1024
 # Max number of characters in a normal Disord message
 MESSAGE_LIMIT = 2000
+# Max number of characters in a Discord nickname
+NICKNAME_CHAR_LIMIT = 32
+# (Lower bound for) Max allowed bytes in a Discord attachment
+ATTACHMENT_BYTE_LIMIT = 8000000
```

Comment:
Starting to think we should move these out to a `constants.py` file. Seeing `config.NICKNAME_CHAR_LIMIT` makes it seem that the character limit is configurable when it's not the case.

---

Edit: I'm just now seeing the title of the PR and realising we might be on the same page lol

Reply:
Haha yeah wow didn't notice I did that. I agree, though since it might affect a lot of files and I have some other PRs I want to do soon that might also affect a lot of files and I consider "higher priority" I think we should merge this for now and make an issue about separating config and constants

Reply:
Actually, we could just rename `config.py` to `constants.py` or `settings.py` or something else since they are effectively fixed at runtime. And some of the discord-pulled constants are in theory something you could WANT to change to something that's not the exact limit imposed by Discord.

I think they have enough semantic similarity and the file is short enough that there's not really an organizational reason to separate them.

Reply:
I do still think of constants as things that are hardcoded, rather than configurable values. `settings.py` sounds good.